ui/src/views/Connection/index.vue

```diff
@@ -414,6 +430,13 @@ const onEvent = (event: string, _data: any) => {
       mittBus.emit('open-setting');
       lunaId.value = _data.lunaId;
       origin.value = _data.origin;
+
+      if (_data.noFileTab) {
+        showTab.value = false;
+      } else {
+        showTab.value = true;
+      }
+
       break;
     case 'file':
       mittBus.emit('open-fileList');
```

Comment:
It appears there is an issue with the `@` keyword usage, as it seems outdated compared to modern JavaScript syntax. Also, some variables like `themeName`, `terminalType` and other components such as `TerminalComponent` might need to be restructured based on their functionality and dependencies.

Here are my suggested changes:

* Replace all occurrences of '@' in your code with '.'
  ```html
  <template v-if="showTab">
```
* Add necessary imports for the components and reset the state when they are initialized:
```javascript
import { ref } from 'vue';

...
const userOptionsRef = ref([])
```

Remember this is just an initial analysis and further reviews could reveal more specific details that might not be immediately apparent at first glance.